### PR TITLE
Auth screen upgrade

### PR DESCRIPTION
**Pull request by rauchp**
if anyone has any suggestions for the default pic lmk

or else we using the most iconic pic in sports history 

Simulator Screen Shot - iPhone 11 - 2019-10-07 at 20 28 09

Simulator Screen Shot - iPhone 11 - 2019-10-07 at 20 28 21

Simulator Screen Shot - iPhone 11 - 2019-10-07 at 20 30 44

### CODE REVIEW

File: civilapp/auth/ProfilePicSetupScreen.js [hunk 1]
```diff
@@ -74,8 +74,15 @@ export default class ProfilePicSetupScreen extends Component {
 
   finishSubmission = () => {
     this.props.navigation.navigate("AuthLoading");
+
   }
 
+  skipSubmission = async () => {
+    const { userId } = this.state
+    const profileImageUrl ='https://i.kym-cdn.com/entries/icons/original/000/016/212/manning.png'
```
Comment (EphemeralX) [inline civilapp/auth/ProfilePicSetupScreen.js]: spoke about this offline

File: civilapp/auth/SignInScreen.js [hunk 2]
```diff
@@ -232,23 +233,27 @@ export default class SignInScreen extends Component {
...
 
-  renderConfirmForm() {
+  renderConfirmForm = () => {
     return (
       <KeyboardAvoidingView
         style={[styles.container, { justifyContent: 'flex-end' }]}
         behavior="padding"
         enabled
       >
+        <TouchableOpacity onPress={() => this.setState({ showConfirmationForm: false, user: {}, userId: '', phoneNumber: 0 })}style={styles.backButtonViewStyle}>
```
Comment (EphemeralX) [inline civilapp/auth/SignInScreen.js]: cody-linter warning: missing space.

File: civilapp/profile/ProfileView.js [hunk 3]
```diff
@@ -70,7 +70,7 @@ class ProfileView extends React.Component {
     thisUserInfo = JSON.parse(thisUserInfo);
 
     //Get user info and set state to render
-    const userId = this.props.navigation.getParam('userId', userInfo.id)
+    const userId = this.props.navigation.getParam('userId', thisUserInfo.id)
```
Comment (EphemeralX) [inline civilapp/profile/ProfileView.js]: made a comment on the last PR; I think thisUserInfo is a little ambiguous what it means, so I think authenticatedUserInfo might make more sense? open to other ideas.

Comment (rauchp) [reply civilapp/profile/ProfileView.js]: nah I fixed it in the other

**Pull request merged**